Source/WebInspectorUI/UserInterface/Base/Utilities.js

```diff
@@ -258,23 +258,6 @@ Object.defineProperty(Set.prototype, "equals",
     }
 });
 
-Object.defineProperty(Set.prototype, "difference",
```

Comment:
Since this PR is about fixing a recent regression, please put the drive-by removal into a separate PR. There could still be fallout from switching to builtin versions of these methods, so I want to ensure that this change receives enough attention (i.e. auditing all the uses)

Reply:
Please post a clean PR with this change before merging ❤️

Reply:
<https://github.com/WebKit/WebKit/pull/42786> (_Web Inspector: use new JS builtin methods instead of custom utility functions_)